dmscripts/helpers/framework_helpers.py

```diff
@@ -111,6 +121,10 @@ def add_supplier_info(client, record):
     return dict(record, supplier=supplier['suppliers'])
 
 
+# TODO: the following functions need to be completely rewritten so that they
+# add full objects rather than non-standard keys
```

Comment:
Given the urgency, we're not going to deal with this now.

Reply:
Yep 👍 This was more of a note to self lol.